src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/TelekineticUtilitySpell.java

```diff
@@ -0,0 +1,116 @@
+                    if (equippedTelekineticUtilitySpell.level.getValue() > highestLevelSpell.level.getValue()) {
+                        highestLevelSpell = equippedTelekineticUtilitySpell;
+                    }
+                }
+                equippedTelekineticUtilitySpells.remove(highestLevelSpell);
+                int reducedManaCost = highestLevelSpell.manaCost;
+                for (TelekineticUtilitySpell equippedTelekineticUtilitySpell : equippedTelekineticUtilitySpells) {
+                    reducedManaCost *= 1 - manaCostReductions.get(equippedTelekineticUtilitySpell.level);
+                }
+                // Will cost as little as 1 mana per second
+                highestLevelSpell.reduceMana(Math.max(reducedManaCost, 1), player);
+            }
+        }
+    }
+
+    @Override
+    public void appendHoverText(ItemStack pStack, @Nullable Level pLevel, List<Component> pTooltipComponents, TooltipFlag pIsAdvanced) {
+        pTooltipComponents.add(new TextComponent("While equipped in utility slot:").withStyle(ChatFormatting.GRAY));
+        pTooltipComponents.add(new TextComponent("  - Gives creative flight").withStyle(ChatFormatting.GRAY));
+        if (this.level.getValue() >= ArmorTier.SINGULARITY.getValue()) {
+            pTooltipComponents.add(new TextComponent("  - Gives elytra flight"));
+        }
+        pTooltipComponents.add(new TextComponent("Costs ").withStyle(ChatFormatting.GRAY)
+                                       // TODO: Color subject to change once mana UI is implemented
+                                       .append(new TextComponent(manaCost + " Mana").withStyle(ChatFormatting.BLUE).withStyle(ChatFormatting.UNDERLINE))
+                                       .append(" per second of flight").withStyle(ChatFormatting.GRAY));
+        pTooltipComponents.add(new TextComponent("Additional copies reduce Mana cost by ").withStyle(ChatFormatting.GRAY)
+                                       .append(new TextComponent((int) (manaCostReductions.get(this.level) * 100) + "%").withStyle(ChatFormatting.BLUE).withStyle(ChatFormatting.UNDERLINE))
+                                       .append(" each"));
+    }
+
+    public static class TelekineticUtilitySpellConfig implements IConfig {
+
+        public IntValue MANA_COST;
+        public final ArmorTier LEVEL;
+
+        private final int manaCost;
+
+        public TelekineticUtilitySpellConfig(int manaCost, ArmorTier level) {
+            this.manaCost = manaCost;
+            this.LEVEL = level;
+        }
+
+        @Override
+        public void build(Builder builder) {
+            builder.push("Telekinetic Utility");
+            builder.push("Tier " + LEVEL.getValue());
+            MANA_COST = builder.comment("Mana cost").defineInRange("mana_cost", manaCost, 0, 10000);
+            builder.pop();
```

Comment:
pop again here; i'm not sure why I don't in the example spell

Reply:
Done.